app/src/main/java/org/mozilla/fenix/theme/ThemeManager.kt

```diff
@@ -76,38 +75,27 @@ abstract class ThemeManager {
         private fun updateLightSystemBars(window: Window, context: Context) {
             if (SDK_INT >= Build.VERSION_CODES.M) {
                 window.statusBarColor = context.getColorFromAttr(android.R.attr.statusBarColor)
-                // This will be addressed on https://github.com/mozilla-mobile/fenix/issues/17808
-                @Suppress("DEPRECATION")
-                window.decorView.systemUiVisibility =
-                    window.decorView.systemUiVisibility or SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
+                getWindowInsetsController(window.decorView)?.isAppearanceLightStatusBars = true
```

Comment:
Maybe outside of the scope of this PR: Would the `WindowInsetsControllerCompat` let us remove the `Build.VERSION_CODES` check if it can handle the right API internally?

Reply:
Yes, I already have that implemented locally, just waiting for this to be merged 😀 .

Reply:
No, because `isAppearanceLightStatusBars` does not have any implementation below M. ANd `isAppearanceLightNavigationBars` does not have implementations below O.

Reply:
I will hoever use _getWindowInsetsController_ extension from AC when https://github.com/mozilla-mobile/android-components/pull/10566 is merged and gets into Fenix.